server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/bucket/ReverseNestedIT.java

```diff
@@ -50,30 +50,52 @@
 
     @Override
     public void setupSuiteScopeCluster() throws Exception {
-        assertAcked(prepareCreate("idx1")
-                .setMapping(
-                        jsonBuilder().startObject().startObject("properties")
-                                .startObject("field1").field("type", "keyword").endObject()
-                                .startObject("alias")
-                                    .field("type", "alias")
-                                    .field("path", "field1")
-                                .endObject()
-                                .startObject("nested1").field("type", "nested").startObject("properties")
-                                    .startObject("field2").field("type", "keyword").endObject()
-                                .endObject().endObject()
-                                .endObject().endObject()
-                ));
-        assertAcked(prepareCreate("idx2")
-                .setMapping(
-                        jsonBuilder().startObject().startObject("properties")
-                                .startObject("nested1").field("type", "nested").startObject("properties")
-                                    .startObject("field1").field("type", "keyword").endObject()
-                                        .startObject("nested2").field("type", "nested").startObject("properties")
-                                            .startObject("field2").field("type", "keyword").endObject()
-                                        .endObject().endObject()
-                                    .endObject().endObject()
-                                .endObject().endObject()
-                )
+        assertAcked(
+            prepareCreate("idx1").setMapping(
```

Comment:
That's rather unfortunate. But I think we can live with it. We should probably try using some other formatting methods.

Reply:
The simplest is probably to assign the builder to a local variable, and use blocks to control the indent. For example:

https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/logstash/src/main/java/org/elasticsearch/xpack/logstash/Logstash.java#L105

I also [added some helper methods to XContentBuilder](https://github.com/elastic/elasticsearch/pull/76464/files#diff-a1d08e385062e43f85712f3d422c77034f04365dfabeb6f24aac6ef033b7a1a3) for simple cases, e.g.

```java
builder.startObject()
	.object("foo", o -> o.field("bar", "baz"))
    .array("fred", a -> { a.value("barney"); a.value("wilma"); })
    .endObject();
```

Reply:
There's probably a few things like this.  My opinion is that, rather than trying to find and fix them in this mess (I still can't believe this is a thirty-THOUSAND line delta), we should merge this as is and then fix those as we find them.  That'll also make it easier to backport.  I'm not intending to backport this PR directly (that would more or less involve re-doing every merge conflict for the past three years), but rather I'm going to just make the same changes to the exclusions file and re-run spotless on 7.x.  So anything we fix here will need to be fixed by hand in that PR also.  Instead, if we fix them in follow up PRs, those should backport (relatively) cleanly.